client/signup/steps/import-from/wordpress/content-chooser/index.tsx

```diff
@@ -36,29 +35,32 @@ export const ContentChooser: React.FunctionComponent< Props > = ( props ) => {
 	 */
 	const showJetpackConnectionBlock = !! fromSite;
 	const [ hasOriginSiteJetpackConnected, setHasOriginSiteJetpackConnected ] = useState( false );
-	const [ isFetchingSite, setIsFetchingSite ] = useState( false );
+	const [ initialFetching, setInitialFetching ] = useState( true );
 
 	/**
 	 ↓ Effects
 	 */
-	useEffect( checkOriginSiteJetpackConnection, [ fromSite ] );
+	useEffect( () => {
+		checkOriginSiteJetpackConnection();
+		const interval = setInterval( checkOriginSiteJetpackConnection, 5000 );
```

Comment:
Must the check be made every 5 seconds or just one time?

Reply:
Yes, the check should be triggered every 5 seconds.
That is why the setInterval function is there.

Reply:
Nice, so it's ok!